MojioSDK/Client/AuthClient.swift

```diff
@@ -306,6 +306,10 @@ public class AuthClient: NSObject, AuthControllerDelegate {
                 failure(response: response.result.value as? NSDictionary)
             }
         }
```

Comment:
If I were to look at the codebase with a fresh set of eyes, I would not know what `#if DEBUG ` meant. To a general person, it means that the build should be in debug mode and -DDBUG is not the first thing to cross my mind. It's possible to spend a lot of time trying to figure out why the warnings aren't showing up when in theory they should.